src/jit/lowerarm64.cpp

```diff
@@ -1051,10 +1054,7 @@ void Lowering::TreeNodeInfoInitPutArgStk(GenTree* argNode, fgArgTabEntryPtr info
     // Do we have a TYP_STRUCT argument (or a GT_LIST), if so it must be a 16-byte pass-by-value struct
     if ((putArgChild->TypeGet() == TYP_STRUCT) || (putArgChild->OperGet() == GT_LIST))
     {
-        // We will use two store instructions that each write a register sized value
-
-        // We must have a multi-reg struct 
-        assert(info->numSlots >= 2);
```

Comment:
How about modifying this assert as follows?

assert(info->numSlots >= 2 && info->numSlots <= MAX_ARG_REG_COUNT)